templates/cc-payment-fields.php

```diff
@@ -1,7 +1,7 @@
 <div class="ginput_complex<?php echo $class_suffix; ?> ginput_container ginput_container_creditcard hps_secure_cc" id="<?php echo $field_id; ?>">
   <div id="HPS_secure_cc">
     <div class="ss-shield"<?php echo $this->get_tabindex(); // added to get rid of GF silly confusion for the menu item being the next index from the first input ?>></div>
-      <div id="gp-error" class="form-group"></div>
+
       <br>
       <div id="secure-submit-card" class="form-group">
```

Comment:
Why are we using the id "secure-submit-card" in two places?

Reply:
@MSmedal I have kept the same id for consistency of payment fields which we usually follow in other plugins as well.